worlds/ror2/__init__.py

```diff
@@ -36,7 +36,7 @@ class RiskOfRainWorld(World):
     location_name_to_id = item_pickups
 
     data_version = 6
```

Comment:
You're adding a new item to your item table, so you should increment this.

```suggestion
    data_version = 7
```

Reply:
I thought this was deprecated?

Reply:
Older clients haven't been dropped completely yet, so we still support it... for now.